pom.xml

```diff
@@ -1343,6 +1357,7 @@
 		<slf4jVersion>1.7.36</slf4jVersion>
 		<log4jVersion>2.25.2</log4jVersion>
 		<owaspCsrfGuardVersion>4.5.0-jakarta</owaspCsrfGuardVersion>
+		<graalVmVersion>25.0.1</graalVmVersion>
```

Comment:
This should probably be `graalJsVersion`. Graal is, for better or worse, a lot of things and the JS engine is just one part.

Reply:
Resolved